react/features/toolbar/functions.web.js

```diff
@@ -0,0 +1,159 @@
+/* @flow */
+
+import SideContainerToggler
+    from '../../../modules/UI/side_pannels/SideContainerToggler';
+
+import defaultToolbarButtons from './defaultToolbarButtons';
+
+declare var AJS: Object;
+declare var interfaceConfig: Object;
+declare var $: Function;
```

Comment:
Let's look into it outside of this PR though. And let's do it sooner rather than later.